### ISSUE TITLE
MONAI version 4 changes norm to N 

### ISSUE THREAD

**Issue by kissievendor** [Project-MONAI/MONAI#1410]
**Describe the bug**
With the update, I can't load existing models anymore, because it expects 'model.0.N.weight' instead of 'model.0.norm.weight'.

**To Reproduce**
model = UNet(
dimensions=3,
in_channels=1, 
out_channels=7,
channels=(32,64,128,256),
strides=(2,2,2),
norm=Norm.BATCH).to(device)

checkpoint = torch.load('experiment.pth')
model.load_state_dict(checkpoint['model_state_dict'])

**Expected behavior**
Model loading and creating predictions. 

**Screenshots**
![image](https://user-images.githubusercontent.com/54887561/103762634-ecf74f80-5018-11eb-9466-f8f9d1950da5.png)

**Environment**

```
![image](https://user-images.githubusercontent.com/54887561/103762833-42336100-5019-11eb-8957-fbefc54f45c3.png)

```

I would like to use some features from version 4, like the new Hausdorff metric. So it would be useful if I could load my original models as well. 

Thank you.

**Comment by wyli** [comment]
thanks for the detailed report, this is also related to https://github.com/Project-MONAI/MONAI/issues/233. it's kind of expected as the project is under active development and we currently don't provide a backward compatibility guarantee. we could create a script to rename the variables if this becomes a frequent request. at the moment please try renaming the variables in the trained models
